MAUI/DigitalGauge/Customize-character-segments.md

```diff
@@ -0,0 +1,268 @@
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="#146CED"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="EightCrossEightDotMatrix"                
+                          CharacterStroke="#146CED"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 360;
+
+            digital.Time = "01-01-18";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 3;
+
+            digital.CharacterSpacing = 10;
+
+            digital.CharacterType = DigitalGaugeCharacterType.EightCrossEightDotMatrix;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize character segment stroke
+
+The values of digital characters color can be customized using the `CharacterStrokeColor` property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <gauge:SfDigitalGauge Text="1 2 3 4 5" StrokeWidth="3" 
+                          HeightRequest="100"  WidthRequest="300"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="#146CED"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SevenSegment" 
+                          CharacterStroke="Purple"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 300;
+
+            digital.Text = "1 2 3 4 5";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 3;
+
+            digital.CharacterType = DigitalGaugeCharacterType.SevenSegment;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.Purple;
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize disabled segment
+
+You can customize the color and opacity of disabled segments using the `DisabledSegmentStroke` and `DisabledSegmentStroke`properties. The width of the digital character value can be customized using the `StrokeWidth` property.
```

Comment:
The width of the digital character value can be customized using the `StrokeWidth` property. - no need